.github/CODEOWNERS

```diff
@@ -4,9 +4,9 @@
 # Here is information about how to configure this file:
 # https://help.github.com/en/articles/about-code-owners
 
-# API and monitoring team will help to review swagger.yml changes.
+# monitoring team will help to review swagger.yml changes.
 #
-http/swagger.yml @influxdata/api @influxdata/monitoring-team
+http/swagger.yml @influxdata/monitoring-team
 
 # Storage code
 /storage/ @influxdata/storage-team-assigner
```

Comment:
I don't see the add for pkger anywhere 🤔

Reply:
derp, lost it just before making the commit